gradle/dependencies.gradle

```diff
@@ -108,7 +108,9 @@ versions += [
   kafka_41: "4.1.1",
   log4j2: "2.25.1",
   // When updating lz4 make sure the compression levels in org.apache.kafka.common.record.CompressionType are still valid
-  lz4: "1.8.0",
+  // https://github.com/apache/kafka/blob/trunk/clients/src/main/java/org/apache/kafka/common/record/CompressionType.java#L73-L74
+  // https://github.com/yawkat/lz4-java/blob/main/src/java/net/jpountz/lz4/LZ4Constants.java#L23-L24
+  lz4: "1.8.1",
```

Comment:
Should we directly bump to 1.10.0?

Reply:
done (https://github.com/apache/kafka/pull/21035/commits/cd8be9e4951627b9f7e9032ba8f86b2075fb7ad4)